### PR TITLE
Makes 'Load More' button disabled if there are no more images

### LINKED ISSUE #526: "Load more" button still clickable even when there are no images to load
**Is your feature request related to a problem? Please describe.**
After a user loads an image and there are no more images left, the link still says 'Load more'.

**Describe the solution you'd like**
I think the link text should change to load more and the link should be disabled so the users don't  keep clicking on the button and nothing happens.

### PR DESCRIPTION
**Pull request by SomtochiAma**
Fixes #526 

[Short description explaining the high-level reason for the pull request]
When there are no more images from the search to be loaded, it disables the button and also changes the text

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: src/components/SearchGridManualLoad.vue [hunk 1]
```diff
@@ -97,6 +99,9 @@ export default {
     _query() {
       return this.$props.query;
     },
+    isFinished() {
+      return this.currentPage * 20 >= this.$store.state.imagesCount;
```
Comment (brenoferreira) [inline src/components/SearchGridManualLoad.vue]: I'm not sure I understood the logic here. Shouldn't it compare `state.images.length` with `state.imagesCount`?

Comment (SomtochiAma) [reply src/components/SearchGridManualLoad.vue]: So for some reason, sometimes there are fewer images in the array than the image Count even when there are no more images to load.
So the "No more Images" never shows and the link still works.
But checking the number of pages multiplied by the default number of images works.

Comment (brenoferreira) [reply src/components/SearchGridManualLoad.vue]: Do you know of a few search terms where this reproduces? This seems like a bug on the API, right @aldenstpage ?

Comment (SomtochiAma) [reply src/components/SearchGridManualLoad.vue]: You can search "meangirls", 
The imageCount is 94 but it returns 85 images

Comment (brenoferreira) [reply src/components/SearchGridManualLoad.vue]: That looks like a bug in the API.  I'll file an issue there and when it's fixed, we can fix it here

Comment (brenoferreira) [reply src/components/SearchGridManualLoad.vue]: Although for now, I think we can fix it using the `page_count` value that comes in the response to the search request as you can see [here](https://api.creativecommons.engineering/image/search?q=meangirls). Saving that on the store and then checking if the `currentPage === pageCount` should be good enough

**Pull request merged**